vizro-core/examples/visual-vocabulary/app.py

```diff
@@ -111,17 +111,11 @@ def make_navlink(chart_group: ChartGroup) -> vm.NavLink:
 
 app = Vizro().build(dashboard)
 app.dash.layout.children.append(
-    html.Div(
-        [
-            html.Div(
-                [
-                    "Made using ",
-                    html.Img(src=get_asset_url("logo.svg"), id="banner", alt="Vizro logo"),
-                    dbc.NavLink("vizro", href="https://github.com/mckinsey/vizro", target="_blank", external_link=True),
-                ],
-                className="anchor-div",
-            ),
-        ],
```

Comment:
@nadijagraca - there were lots of redundant html.divs, probably from some legacy code. It's always good to simplify where possible. Putting the dbc.NavLink around the entire container also ensures that the entire container is reactive, not only parts of the container :)